core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/HealthMonitor.scala

```diff
@@ -285,6 +297,34 @@ class HealthMonitor private (val slickDataSource: SlickDataSource,
     samDAO.getStatus()
   }
 
+  private def checkBPM: Future[SubsystemStatus] = {
+    logger.debug("Checking Billing Profile Manager...")
+    Future(
+      SubsystemStatus(
+        billingProfileManagerDAO.getStatus().isOk,
+        Option(billingProfileManagerDAO.getStatus().getSystems).map { subSystemStatuses =>
```

Comment:
Move the option into the `for` comprehention (use `toList` within to make the types line up)